packages/language/src/syntax-tree/ast.ts

```diff
@@ -1965,13 +1965,15 @@ export interface ReferenceItem extends AstNode {
   kind: SyntaxKind.ReferenceItem;
   ref: Reference<NamedElement> | null;
   dimensions: Dimensions | null;
+  token: Token | null;
```

Comment:
Suggestion: Can we reuse the `ref.token` value instead? I don't think it's necessary to store the reference item token twice.